deps/wazuh_testing/wazuh_testing/tools/wazuh_manager.py

```diff
@@ -32,11 +38,30 @@ def remove_agents(agents_id, remove_type='wazuhdb'):
                 'older_than': '0s'
             }
             url = f"{api_details['base_url']}/agents"
-            response = requests.delete(url, headers=api_details['auth_headers'], params=payload, verify=False)
+            response = requests.delete(
+                url, headers=api_details['auth_headers'], params=payload, verify=False)
```

Comment:
Please check the indentation.

Reply:
Fixed on commit [`a915da5`](https://github.com/wazuh/wazuh-qa/pull/3662/commits/a915da5cf02906ea8e70673489fb69f3bde08639)